src/components/ResourceList/components/FilterControl/types.ts

```diff
@@ -1,5 +1,5 @@
-import {SelectOption} from '../../../Select';
-import {TextFieldProps} from '../../../TextField';
+import type {SelectOption} from '../../../Select';
+import type {TextFieldProps} from '../../../TextField';
```

Comment:
There are a few imports that come from the Component name. Should we move them to types?

Reply:
talked about this on slack, but short answer is: No: `TextField` in this case is the public API of the component